src/SFA.DAS.ApplyService.Web/Views/Application/Pages/Index.cshtml

```diff
@@ -216,22 +218,41 @@
 
                         @if (!Model.HideCTA)
                         {
-                            <button type="submit" class="govuk-button">@Model.CTAButtonText</button>
+                            if (Model.PageOfAnswers != null && Model.PageOfAnswers.Count > 0 && Model.PageOfAnswers.First().Answers.Count(x => x.Value.ToLower().Contains(".pdf"))>0)
+                            {
+                                <a class="govuk-button" href="@Url.Action("TaskList", "RoatpApplication", new { Model.ApplicationId })">@Model.CTAButtonText</a>
+                            }
+                            else
+                            {
+                                <button type="submit" class="govuk-button">@Model.CTAButtonText</button>
+
+                            }
                         }
 
                         @foreach (var linksConfiguration in Model.LinksConfiguration)
                         {
-
+                            var first = true;
                             @foreach (var link in linksConfiguration.Links)
+                            {
+                                if (first && Model.SectionId != 1)
                                 {
-                                <p class="govuk-body govuk-!-margin-bottom-2">
-                                    <a href="@Url.Action(link.Action, link.Controller, new { Model.ApplicationId, sequenceId = Model.SequenceId, sectionId = Model.SectionId, pageId = Model.PageId})">@link.Caption</a>
-                                </p>
+                                    <p class="govuk-body skipbutton" id="skipbutton">
```

Comment:
I think we should raise a tech debt story to restructure this so that the css class is in links configuration, rather than hard-coding the first link if section id != 1

Reply:
I'll do that